src/events.zig

```diff
@@ -439,6 +442,48 @@ pub const Key = struct {
     repeat: bool,
 };
 
+/// Mouse button event structure
+///
+/// ## Version
+/// This struct is available since SDL 3.2.0.
+///
+/// ## Code Examples
+/// ```zig
+///                 see https://wiki.libsdl.org/SDL3/SDL_MouseButtonFlags
```

Comment:
This link should not be in an embedded code block. Also is the link needed? Inline-docs for the mouse subsystem should cover the flags well.